src/main/java/com/googlecode/aviator/lexer/ExpressionLexer.java

```diff
@@ -180,6 +180,8 @@ public int getCurrentIndex() {
       StringBuffer sb = new StringBuffer();
       int startIndex = this.iterator.getIndex();
       long lval = 0L;
+      long overflowFlag = Long.MAX_VALUE / 10;
```

Comment:
I think maybe we can make `overflowFlag` and `overflowSingle` as constant values insteadof calculating them everytime.